containers/datalab/Dockerfile.in

```diff
@@ -36,5 +36,7 @@ ENV EXPERIMENTAL_KERNEL_GATEWAY_URL=""
 # Startup
 ENV DATALAB_VERSION _version_
 ENV DATALAB_COMMIT _commit_
-ENTRYPOINT [ "/datalab/run.sh" ]
+ENV DEPLOY_VM "false"
```

Comment:
This looks like a "hidden" feature that will almost never be used and exercised. IMO it is better to remove it so we don't need to maintain it. 

Maybe we can also provide a demo deploy.sh to help user create the VM.